crates/linker/src/lib.rs

```diff
@@ -2098,26 +2091,22 @@ fn surgery_elf_and_macho(
     if verbose || time {
         println!("\nTimings");
         report_timing("Loading Metadata", loading_metadata_duration);
-        report_timing("Application Parsing", app_parsing_duration);
         report_timing("Loading and mmap-ing", load_and_mmap_duration);
         report_timing("Output Generation", out_gen_duration);
         report_timing("Flushing Data to Disk", flushing_data_duration);
-        report_timing(
-            "Other",
-            total_duration
-                - loading_metadata_duration
-                - app_parsing_duration
-                - load_and_mmap_duration
-                - out_gen_duration
-                - flushing_data_duration,
-        );
+
+        let sum = loading_metadata_duration
+            + load_and_mmap_duration
+            + out_gen_duration
+            + flushing_data_duration;
+
+        report_timing("Other", total_duration.saturating_sub(sum));
```

Comment:
somehow the total duration was smaller than the sum of the other durations. I'm guessing its because the clock is not that precise? I went over the code and I don't think there are mistakes in when the timings start/stop.

Reply:
Are we using the wrong clock, I know there are multiple with different setups? Othewise, rounding error maybe? Not really sure.